flake.nix

```diff
@@ -90,6 +90,7 @@
             stableToolchain
 
             cargo-edit
+            cargo-sort
```

Comment:
It would be nice to note the invocation (e.g. `cargo sort -w` locally, `cargo sort -w -c` for CI) somewhere. Maybe in README.md?